inc/test_case.json

```diff
@@ -0,0 +1,128 @@
+{
+    "Test Case":
+    [
+        {
+            "Input Array": [4, 5],
+            "Range Output" : ["4-5, 2"]
+        },
+        {
+            "Input Array": [3, 3, 5, 4, 10, 11, 12],
+            "Range Output": ["3-5, 4", "10-12, 3"]
+        },
+        {
+            "Input Array": [1,1,1,1,1],
+            "Range Output": ["1, 5"]
+        },
+        {
+            "Input Array": [1, 2, 3, 6],
+            "Range Output": ["1-3, 3", "6, 1"]
+        },
+        {
+            "Input Array": [1, 3, 5, 7, 9],
+            "Range Output": ["1, 1", "3, 1", "5, 1", "7, 1", "9, 1"]
+        },
+        {
+            "Input Array": [1],
+            "Range Output": ["1, 1"]
+        },
+        {
+            "Input Array": [],
+            "Range Output": []
+        }
+    ],
+    "Test Case A2D":
```

Comment:
Consider making separate tests in a different file - that way, different people can add tests without interfering